Assets/MirageXR/ContentTypes/Audio/Editor/Mobile/AudioEditorView.cs

```diff
@@ -23,7 +24,7 @@ public class IntHolder : ObjectHolder<int> { }
     public override ContentType editorForType => ContentType.AUDIO;
 
     [SerializeField] private Button _btnAudioSettings;
-    [SerializeField] private Button _btnCancel;
+    [SerializeField] private Button _btnMicRecording1, _btnMicRecording2;
```

Comment:
```suggestion
    [SerializeField] private Button _btnMicRecording1; 
    [SerializeField] private Button _btnMicRecording2;
```

Is it possible to give them a more specific name without numbers?

Reply:
fixed